modules/CmpdReg/src/server/routes/CmpdRegRoutes.coffee

```diff
@@ -1,5 +1,4 @@
 exports.setupAPIRoutes = (app) ->
-	app.post '/api/cmpdReg', exports.postAssignedProperties
```

Comment:
exports.postAssignedProperties doesn't exist and express 4 was throwing and error so I needed to remove it.